src/pages/jj/config/api.js

```diff
@@ -0,0 +1,3 @@
+const APIKEY = 'http://localhost:3000/data/jj/Data.json';
```

Comment:
지금은 mock data를 가져오는 단순한 요청이라 괜찮지만 보통 아래와 같이 합니다
```suggestion
const BASE_URL = 'http://localhost:3000';

const DATA_URL =`${BASE_URL}/data/jj/Data.json`
```

Reply:
> 지금은 mock data를 가져오는 단순한 요청이라 괜찮지만 보통 아래와 같이 합니다

앗 그렇군요 ! 감사합니다ㅠ!!